tests/unit/taqueria-utils/taqueria-utils.test.ts

```diff
@@ -0,0 +1,142 @@
+import { assertEquals, assertRejects, unreachable } from "https://deno.land/std@0.121.0/testing/asserts.ts";
+import {fork, promise} from 'https://cdn.skypack.dev/fluture';
+import {
+    decodeJson,
+    isTaqError,
+    log,
+    mkdir,
+    renderTemplate,
+    writeTextFile
+} from "../../../taqueria-utils/taqueria-utils.ts";
+import chai from "https://cdn.skypack.dev/chai@4.3.4?dts";
+import { exists} from "https://deno.land/std/fs/mod.ts";
+import {TaqError} from "../../../taqueria-utils/taqueria-utils-types.ts";
+
+const testValidJson = '{"test": "testPayload"}';
+const testInvalidJson = '{"test": testPayload}';
+
+Deno.test("Positive scenario test for {decodeJson} function", () => {
+    const result = decodeJson(testValidJson);
+    const decodedJson = { test: "testPayload"};
+    const assertSuccess = (testJsonOutput: any) => assertEquals(testJsonOutput, decodedJson)
+    const assertUnreachable = () => unreachable();
+    fork (assertUnreachable) (assertSuccess) (result);
+});
+
+// TODO: Michael to help with solution for return ()
```

Comment:
Ivan, mind creating an issue for this and adding it to the todo mentioned above please?